compiler/record-minmax/src/RecordMinMax.cpp

```diff
@@ -37,37 +37,20 @@ namespace
 {
 
 /**
- * @brief  clipAndAverage does the following things
- *         (1) Discard the top/bottom clip_ratio (%) values from vector
- *             Ex: top_clip_ratio = 5, bottom_clip_ratio = 3
- *                 discard top 5 % values and bottom 3 % values from vector
- *         (2) Calculate the average of remaining values
+ * @brief  getMovingAverage calculates the weighted moving average of input vector
+ *         The initial value is the first element of the vector
  */
-float clipAndAverage(std::vector<float> &vector, float top_clip_ratio, float bottom_clip_ratio)
+float getMovingAverage(std::vector<float> &vector, float alpha = 0.9)
```

Comment:
We usually didn't use giving default values like `float alpha = 0.9`. Can you remove `= 0.9` and set this to callers ?